### PR TITLE
Move app into subfolder to prep for mono repo

### PR DESCRIPTION
**Pull request by theramis**
I couldn't get husky install to work on my windows pc. 

So will need someone to add that please.

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -20,6 +20,6 @@ jobs:
           node-version: ${{ env.NODE_VERSION }}
           cache: "npm"
       - name: Install dependencies
-        run: npm install
+        run: npm ci
```
Comment (andycarrell) [inline .github/workflows/lint.yml:23]:
```suggestion
        run: npm install
```

I _think_ we wanna keep this as install - `ci` is important for the build process, but not really for eslint, where we're happy for the "other stuff" to happen when we install

If we want to verify that `npm ci && npm run build` work as expected I think it's valuable to add that as an additional workflow. For example, at work I setup:

```
- name: Install dependencies - early feedback on production install
  run: npm ci --production
- name: Build
  run: npm run build
```

File: app/pages/_document.tsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line @next/next/no-document-import-in-page
```
Comment (andycarrell) [inline app/pages/_document.tsx:1]:
```suggestion
```

Believe no longer necessary

**Comment by andycarrell** [comment]
@theramis for husky:

https://typicode.github.io/husky/#/?id=does-it-work-on-windows

**Comment by jishaal** [comment]
Move to WSL bro 😏

**Review by andycarrell — APPROVED** [review]

**Pull request merged**